packages/dialtone-css/lib/build/less/utilities/sizing.less

```diff
@@ -136,10 +136,16 @@
 .d-w100vw { width: 100vw !important; }
 .d-w-auto { width: auto !important; }
 .d-w-unset { width: unset !important; }
+.d-w-fit { width: fit-content !important; }
```

Comment:
Hmm.. Let's use the whole word for these. `.d-w-fit-content` instead of `d-w-fit` and so on.

Reply:
Might as well add these to height as well.